Backend/controllers/CaseInvestigationController.js

```diff
@@ -1,12 +1,36 @@
-const { CaseInvestigation } = require("../models/CaseInvestigation.js");
+const CaseInvestigationModels = require("../models/CaseInvestigation.js");
 const mongoose = require("mongoose")
 const _ = require("lodash");
-const { Patient } = require("../models/Patient.js");
-const { SymptomUser } = require("../models/SymptomUser");
-const { User } = require("../models/UserModel.js");
+const PatientModels = require("../models/Patient.js");
+const SymptomUserModels = require("../models/SymptomUser");
+const UserModels = require("../models/UserModel.js");
+
+// check if demo=true in request
+let demo_or_real_db = (query) => {
+    if (query.demo && query.demo == "true"){
+        return {
+            Patient: PatientModels.PatientDemo,
+            SymptomUser: SymptomUserModels.DemoSymptomUser,
+            User: UserModels.DemoUser,
+            CaseInvestigation: CaseInvestigationModels.CaseInvestigationDemo,
+            Name: "Demo", // for populate queries
```

Comment:
Shouldnt this be "Demo " with a space at the end since the model I believe is saved as "Demo User"

Reply:
There's sometimes "Demo " at the start (like in user case) and sometimes there is "Demo" at the end (like in patient or patientLog)